src/custom/hooks/useWalletInfo.ts

```diff
@@ -11,7 +11,7 @@ import { gnosisSafeAtom } from 'state/gnosisSafe/atoms'
 import { useMemo } from 'react'
 
 const GNOSIS_SAFE_APP_NAME = 'Gnosis Safe App'
-const SAFE_ICON_URL = 'https://apps.gnosis-safe.io/wallet-connect/favicon.ico'
+const SAFE_ICON_URL = 'https://app.safe.global/favicon.ico'
```

Comment:
This change has no effect AFAICT.
The favicon loaded is still the former.
Maybe it's coming from somewhere else?

Reply:
Does work when opening the app inside the Safe, so maybe that favicon comes from somewhere else, like an outdated library